conwaylib/Game.cs

```diff
@@ -10,7 +10,8 @@ namespace ConwayLib
   /// </summary>
   public sealed class Game
   {
-    private readonly HashSet<byte[]> mHistory = new HashSet<byte[]>();
+    private HashComparer mHashComparer;
```

Comment:
I dont think you need this as a field, can just be a local variable in the constructor